cabbage-roadmap-2022-1/README.md

```diff
@@ -0,0 +1,136 @@
+# Team Cabbage Roadmap 2022
+
+This RFC describes the roadmap for Team Cabbage for the first part of 2022.
+
+## Background
+
+Since the formation of Team Cabbage just after the Croatia onsite (~September 2021), up to the start of 2022, Team Cabbage's overall focus has been on getting started as a team.
+
+This involved the usual work for team formation, including defining Jobs To Be Done and overall mission statement. We had a number of issues to clean up or sort out, such as dissolving Team Halo, postmortems, and app upgrades. We also updated the monitoring and alerting for our apps.
+
+Finally, we continued improving our support for shipping Kong as a Managed App, and shipped the AWS LB Controller as a Managed App. 
+
+## Jobs To Be Done
+
+The Jobs To Be Done for Team Cabbage can be summarised as follows:
+
+- Exposing services running on a Kubernetes cluster
+- Routing, observing, and securing traffic into, inside, and out of Kubernetes clusters
+- Managing API microservices on Kubernetes clusters
+
+These describe the tasks that users want to do, and help frame the problem space the team works in.
+
+There is a lot of overlap between these jobs, and it is important to remember that we want to provide one story to customers - i.e: providing a turn-key traffic solution for Fortune 500 customers.
+
+## Discovery
+
+Before moving to the more concrete opportunities, it's useful to cover more long-term patterns and use cases we're thinking about. These are items that are not yet ready for us to tackle, but are going to form the basis for our product discovery over the new few months, and will likely form part of our roadmap in the future.
+
+We also need to make sure that we take the current strategic move to Cluster API into account.
+
+### Gateway API
+
+See the [upstream project](https://gateway-api.sigs.k8s.io/).
+
+Gateway API can be described as a possible future replacement for the current [Ingress API object](https://kubernetes.io/docs/concepts/services-networking/ingress/), aiming to provide a more well-rounded API for describing service networking in Kubernetes.
```

Comment:
I think we could mention here already that one of the main use cases is supporting better multi-tenancy and separation of concerns within the use case of exposing services.

Reply:
Added a bit lower